src/globals.d.ts

```diff
@@ -26,6 +26,54 @@ declare namespace Chai {
 }
 /* tslint:enable */
 
+// jsonschema declarations
+declare interface Schema {
```

Comment:
Did you write this def from scratch? If you got it from somewhere, please add the source above it in a comment.